tests/foreman/cli/test_repositories.py

```diff
@@ -0,0 +1,46 @@
+"""Test module for Repositories CLI.
+
+:Requirement: Repository
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: Repositories
+
+:team: Phoenix-content
+
+:TestType: Functional
+
+:CaseImportance: Critical
+
+:Upstream: No
+"""
+import pytest
+
+
+@pytest.mark.rhel_ver_list([7, 8, 9])
+def test_positive_custom_products_disabled_by_default(
+    setup_content,
+    default_location,
+    rhel_contenthost,
+    target_sat,
+):
+    """Verify that custom products should be disabled by default for content hosts
+
+    :id: ba237e11-3b41-49e3-94b3-63e1f404d9e5
+
+    :steps:
+        1. Create custom product and upload repository
+        2. Attach to activation key
+        3. Register Host
+        4. Assert that custom proudcts are disabled by default
+
+    :expectedresults: Custom products should be disabled by default. "Enabled: 0"
+    """
+    ak, org, _ = setup_content
+    client = rhel_contenthost
+    client.register(org, default_location, ak.name, target_sat)
+    assert client.subscribed
+    product_details = rhel_contenthost.run('subscription-manager repos --list')
```

Comment:
It would be nice to validate custom products are is disabled on AK itself before Host, wdyt?

Reply:
this one just tests host. but I added it in the UI test

Reply:
product_content only shows subscriptions attached under an activation key. With sca being enabled and subscriptions not being attached, asserting on the ak will only return an empty list. Asserting the Content Host only should be fine here. plus we are already testing ak in the UI.